module-1/cdk/lib/Module1Construct.ts

```diff
@@ -0,0 +1,74 @@
+import cloudfront = require("@aws-cdk/aws-cloudfront");
+import iam = require("@aws-cdk/aws-iam");
+import s3 = require("@aws-cdk/aws-s3");
+import s3deploy = require("@aws-cdk/aws-s3-deployment");
+import cdk = require("@aws-cdk/cdk");
+
```

Comment:
The README file models this as a `Stack`...

Reply:
Changed whilst you were reviewing.